kyo-os-lib/shared/src/main/scala/kyo/os.scala

```diff
@@ -0,0 +1,213 @@
+package kyo
+
+import java.io.*
+import java.lang.Process as JProcess
+import java.lang.ProcessBuilder.Redirect
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Path
+import java.util.concurrent.TimeUnit
+import kyo.*
+import scala.jdk.CollectionConverters.*
+
+sealed trait Command:
+    self =>
+    import Command.*
+
+    /** Spawns a new process executing this command
+      */
+    def spawn: Process < (IOs)
+
+    /** Spawns a new process and returns its output as a String
+      */
+    def string: String < (IOs) =
+        stream.map { inputStream =>
+            new String(inputStream.readAllBytes())
+        }
+
+    /** Spawns a new process and returns its output as a Stream
+      */
+    def stream =
+        spawn.map(_.stdout)
+
+    def exitValue =
+        spawn.map(_.exitValue)
+
+    def waitFor =
+        spawn.map(_.waitFor)
+
+    def waitFor(timeout: Long, unit: TimeUnit) =
+        spawn.map(_.waitFor(timeout, unit))
+
+    def pipe(that: Command): Command =
+        Command.Piped(self.flatten ++ that.flatten).simplify
+
+    def >>(that: Command): Command =
```

Comment:
We typically avoid symbolic operators in Kyo. What do you think about `andThen`?